src/utils.ts

```diff
@@ -13,3 +16,65 @@ export const aestheticBaseAttributes: Pick<Nft, 'aestheticRank' | 'aestheticRank
     timesWon: 0,
     aestheticRankReliability: 0,
 };
+
+/**
+ * A mapping of all PopRank supported chains to all
+ * of our supported marketplaces.
+ *
+ * The first marketplace under a chain is considered the default.
+ */
+export const chainMarketplaceMap: ChainMarketplaceMap = {
+    ethereum: {
+        opensea: {
+            name: "OpenSea",
+            nftUrlFormatter: (address?: string, id?: string) => `${OPENSEA_BASE_NFTS_URL}/${address}/${id}`,
```

Comment:
did we want these params to be optional?

Reply:
For solana NFTs, all MagicEden needs is the token's mint account address (we treat this as its ID), as for solana each NFT has its own unique mint account, so the address is unused.

So i guess our options are:
1. Leave as is
2. Make `id` mandatory, as so far all chains have an ID
3. Make both mandatory, and when we're formatting solana NFT's links, we can pass in the collection's on chain address, knowing it'll be unused

I think i lean towards 2.

Reply:
resolved offline